### ISSUE TITLE
'no-global-jquery' flags false positive when Ember.$ assigned to a var

### ISSUE THREAD

**Issue by nlfurniss** [ember-best-practices/eslint-plugin-ember-best-practices#85]
```js
const $ = Ember.$;
$.whatever(); // flags '$'
```
